### DISCUSSION TITLE
节点自动更新设置项 能否增加 每小时 之类的设置？

### DISCUSSION THREAD

**Discussion by ATang007ZH** [immortalwrt/homeproxy#253] [Q&A] [answered]
节点自动更新设置项，现在只能每天定点更新。
感觉时间间隔太长了，如果刚更新完节点就失效了就要一天后才自动更新。
能否增加 每小时 之类的设置？

看了计划任务，有相关设置项，但是自己改了或者新增之后，自动更新一次就恢复原样了(新增的也被删掉了）。

**Comment by 1715173329** [comment, marked as answer]
不支持 不会支持。这种垃圾机场没有使用的必要
想要自定义时间可以自己改代码

**Comment by ATang007ZH** [comment]
> 节点自动更新设置项，现在只能每天定点更新。 感觉时间间隔太长了，如果刚更新完节点就失效了就要一天后才自动更新。 能否增加 每小时 之类的设置？
> 
> 看了计划任务，有相关设置项，但是自己改了或者新增之后，自动更新一次就恢复原样了(新增的也被删掉了）。

计划任务被删的问题 源码已经修正了，用12月以后的最新源码就没有问题了。


自己写了个定时检查更新命令放到计划任务里了。
检查节点有更新，才执行更新命令。
对节点不定时更新的来说，省得每次都要手动操作，终于安心了。
